src/engine/world/room.h

```diff
@@ -103,7 +106,10 @@ struct Room
 
   std::shared_ptr<render::scene::Node> node = nullptr;
   std::vector<gslu::nn_shared<render::scene::Node>> sceneryNodes{};
-
+  
+  std::map<uint8_t,std::shared_ptr<render::scene::Node>> dustCache{};
```

Comment:
```suggestion
  std::map<uint8_t, std::shared_ptr<render::scene::Node>> dustCache{};
```
Also general guideline: space after commas.